modules/RepConvGRC.js

```diff
@@ -1711,6 +1711,10 @@ function _RepConvGRC() {
             var website = getStatsLink(what, who_id, who_name);
             RepConvNode.append($('<iframe/>', {'src': website, 'style': 'width: 1020px; height: 580px;'}));
             Layout.dialogWindow.open(RepConvNode.html().toString(), RepConvTool.GetLabel('STATS.'+what.toUpperCase()), 1020, 580, null, false).setPosition("center", "center");
+        } else if (RepConv.active.statsGRCL == 'grepolife'  && (["ru", "us", "de", "en", "fr", "es", "br", "pl", "nl", "it", "pt", "ro"].indexOf(Game.world_id.slice(0, 2)) >= 0)) {
+            var website = getStatsLink(what, who_id, who_name);
+            RepConvNode.append($('<iframe/>', {'src': website, 'style': 'width: 1020px; height: 580px;'}));
+            Layout.dialogWindow.open(RepConvNode.html().toString(), RepConvTool.GetLabel('STATS.'+what.toUpperCase()), 1020, 580, null, false).setPosition("center", "center");
```

Comment:
Not available for sites containing advertisements.

Reply:
Not really sure what's not available since it does work fine for me...

Reply:
@FivosM: surely, it will work, BUT since statistics are being loaded to an in-game windows, then if the source contains advertisements, they will be loaded too. That is not allowed by InnoGames and will make GRCRT disallowed - not what we want :grin: 

I had also requested the list of supported worlds to be dynamic - having to update the script whenever new world on any market gets added to grepolife is not something we would call "a good practice" :wink:

Reply:
@tomaski it would be better if grepolife is not supported. Too many obstacles for things that GRCRT's website can do too.

Reply:
@FivosM we're with @Potusek totally NOT against adding another statistics sources to our script and are open for improvements, but all the changes we implement do need to follow the rules set by InnoGames...